code_examples/workshop/verify.ts

```diff
@@ -14,19 +16,27 @@ function getChallenge(): string {
 
 // verifies validity, ownership & attestation
 async function verifyPresentation(
+  api: ApiPromise,
   presentation: Kilt.ICredentialPresentation,
   challenge: string
 ): Promise<boolean> {
   try {
     await Kilt.Credential.verifyPresentation(presentation, { challenge })
+
+    const encodedAttestationInfo = await api.query.attestation.attestations(
+      presentation.rootHash
+    )
+    const attestationInfo = Kilt.Attestation.fromChain(
+      encodedAttestationInfo,
+      presentation.rootHash
+    )
```

Comment:
Is it more readable this way?
```suggestion
    const attestationInfo = Kilt.Attestation.fromChain(
      await api.query.attestation.attestations(presentation.rootHash),
      presentation.rootHash
    )
```

Reply:
Fixed in https://github.com/KILTprotocol/docs/pull/156/commits/17cd8e70e736129f03d0e5788336e4d596fe5d20.